routers/org/home.go

```diff
@@ -85,7 +85,6 @@ func Home(ctx *context.Context) {
 		OrderBy:            orderBy,
 		Private:            ctx.IsSigned,
 		Actor:              ctx.User,
-		IsProfile:          true,
```

Comment:
unrelated change ?

Reply:
No it's not unrelated, that option was not needed anymore